### DISCUSSION TITLE
Tips for loading the geodataframe faster?

### DISCUSSION THREAD

**Discussion by atpendry** [developmentseed/lonboard#145] [Q&A]
Really excited to try out this tool. Any tips on how to load your geodataframe quickly? I am attempting to load +1million polygons but have been watching it for over an hour. Is this a limitation of using an Esri service?

Is there a file format/ geometry type that works best for testing out this module?
![Snag_70b257](https://github.com/developmentseed/lonboard/assets/83019174/20a66944-24f2-457e-99d9-4f6204d29d71)

**Comment by kylebarron** [comment]
I'm not familiar with Esri libraries. In general, the fastest file format to read and write GeoDataFrames is GeoParquet, which you can do with [read_parquet](https://geopandas.org/en/stable/docs/reference/api/geopandas.read_parquet.html#geopandas.read_parquet) and [to_parquet](https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoDataFrame.to_parquet.html)
